lib/mqtt_abstraction_impl.cpp

```diff
@@ -250,19 +254,14 @@ void MQTTAbstractionImpl::unregister_handler(const std::string& topic, const Tok
         this->handlers.erase(topic);
         // TODO(kai): should we throw/log an error if we are not connected?
         if (this->mqtt_is_connected) {
-            EVLOG(debug) << "Unsubscribing from " << topic;
+            EVLOG(debug) << fmt::format("Unsubscribing from {}", topic);
             this->unsubscribe(topic);
         }
     }
 
-    std::ostringstream oss;
-    oss << "#handler[" << topic << "] = ";
-    if (this->handlers.count(topic) == 0) {
-        oss << "None";
-    } else {
-        oss << this->handlers[topic].size();
-    }
-    EVLOG(debug) << oss.str();
+    const std::string handler_count =
+        (this->handlers.count(topic) == 0) ? "None" : std::to_string(this->handlers[topic].size());
```

Comment:
Usually I'm not the biggest fan of this short if syntax, but here I would say it's acceptable.

Reply:
From a performance perspective it is the right thing to do, but from the readability - it's ok ...
one could write a lambda or helper function for it to make it more readable, but on the other hand it is just that simple and used only once ..